### PR TITLE
Remove workaround for missing patterns

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
## Ticket

- https://progress.opensuse.org/issues/25418

## Verification run

- It has to failed, that is expected: http://copland.arch.suse.de/tests/1498#step/change_desktop/2

### CODE REVIEW

**Comment by SergioAtSUSE** [comment]
New verification run for **installation_overview_before** and **change_desktop**

- http://copland.arch.suse.de/tests/1499#step/change_desktop/2

**Comment by SergioAtSUSE** [comment]
Verification run for **select_patterns_and_packages**

- http://copland.arch.suse.de/tests/1501#step/select_patterns_and_packages/23

File: tests/installation/installation_overview.pm [hunk 1]
```diff
@@ -62,13 +59,7 @@ sub run {
         assert_screen 'autoyast_removed';
     }
 
-    if (check_screen('manual-intervention', 0)) {
-        $self->deal_with_dependency_issues;
```
Comment (coolo) [inline tests/installation/installation_overview.pm]: I think these are still handy for update scenarios. I just wanted broken patterns to be fatal.

**Comment by SergioAtSUSE** [comment]
New verification run:

- installation_overview_before and change_desktop http://copland.arch.suse.de/tests/1502#step/change_desktop/2
- select_patterns_and_packages http://copland.arch.suse.de/tests/1503#step/select_patterns_and_packages/24

**Pull request merged**